openrndr-core/src/main/kotlin/org/openrndr/shape/LineSegment.kt

```diff
@@ -43,15 +48,16 @@ data class LineSegment(val start: Vector2, val end: Vector2) : LinearType<LineSe
     }
 
     /**
-     * find distance between the query point and the point on the line segment that is nearest to the query point
-     * @param query the query point
+     * Find shortest distance to the [LineSegment] from given point.
+     * @param query The query point.
      */
     fun distance(query: Vector2): Double = nearest(query).distanceTo(query)
 
     /**
-     * calculate a sub line segment from t0 to t1
-     * @param t0 t-parameter value at start of line segment
-     * @param t1 t-parameter value at end of line segment
+     * Calculates a new subsegment between two [t](https://pomax.github.io/bezierinfo/#explanation)
+     * values of the [LineSegment].
+     * @param t0 The [t](https://pomax.github.io/bezierinfo/#explanation) value marking the start of the subsegment, between `0.0` and `1.0`.
+     * @param t1 The [t](https://pomax.github.io/bezierinfo/#explanation) value marking the end of the subsegment, between `0.0` and `1.0`.
      */
     fun sub(t0: Double, t1: Double): LineSegment {
```

Comment:
I don't think we need to reference the explanation of what the `t` value is on every method. I think that should be done in the class description.

I'd rather see the range [0.0, 1.0] as the first information

Reply:
Another option is to leave the link as it is and later replace it by a link in the guide showing a photo of Mr. t.

Reply:
I'm sort of okay with the link on every `t`-related comment, it gets hidden when rendered so it's just a single letter as a link.

Reply:
I've gone ahead and removed ~half of the links, so that the link is only provided the first time *t* is mentioned.